kodilla-stream/src/main/java/com/kodilla/stream/immutable/ForumUser.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.stream.immutable;
+
+public final class ForumUser {
+    private final String username;
+    private final String realName;
+
+    public ForumUser(String final username, String final realName) {
```

Comment:
Nie musisz używać `final` przy argumentach, aby klasa była niemutowalna - ale jeśli chcesz to robić, upewnij się, jakie jest tego znaczenie (możemy o tym porozmawiać na video).
Wg przyjętej konwencji najpierw pisze się słowo `final`, a potem typ, czyli tutaj np. `final String username`.